### PR TITLE
Add abstract submission form to UGM page and blog

### PR DESCRIPTION
**Pull request by jennaswa**

### CODE REVIEW

**Review by micaela-matta — APPROVED** [review]
LGTM

File: _posts/2024-02-01-abstracts-ugm2024 [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+layout: post
+title: Call for Abstracts for MDAnalysis UGM 2024
+---
+
+<img
+src="https://github.com/MDAnalysis/branding/blob/main/logos/banners/mdanalysis-banner-UGM-white-1280x144.png"
+title="MDAnalysis UGM Banner" alt="MDAnalysis UGM Banner"
+style="float: left; height: 8em; " />
+
+We are now [accepting abstracts][abstracts] for the [2024 MDAnalysis UGM][ugmPage], taking place August 21-23, 2024, in London, UK at [King’s College London](https://www.kcl.ac.uk/visit/bush-house). Abstracts are welcomed from all areas relevant to MDAnalysis's work, from scientific applications (e.g., biomolecular simulations, soft matter and materials science, drug discovery and more) to the use and development of open source software and tools for molecular simulations data analysis. When submitting your abstract, you may indicate whether you prefer to give a 15 minute talk, 5 minute lighting talk, or poster presentation.
```
Comment (orbeckst) [inline _posts/2024-02-01-abstracts-ugm2024:11]: Write out UGM somewhere... not everyone knows the abbrev
```suggestion
We are now [accepting abstracts][abstracts] for the [2024 MDAnalysis UGM][ugmPage] (User Group Meeting), taking place August 21-23, 2024, in London, UK at [King’s College London](https://www.kcl.ac.uk/visit/bush-house). Abstracts are welcomed from all areas relevant to MDAnalysis's work, from scientific applications (e.g., biomolecular simulations, soft matter and materials science, drug discovery and more) to the use and development of open source software and tools for molecular simulations data analysis. When submitting your abstract, you may indicate whether you prefer to give a 15 minute talk, 5 minute lighting talk, or poster presentation.
```

File: _posts/2024-02-01-abstracts-ugm2024 [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+MDAnalysis strives to be a diverse and welcoming community for all. While the event will be free to attend, a limited number of travel bursaries are available to enable attendance for those facing financial barriers. If you would like to apply, please follow the prompts on the[abstract submission form][abstracts].
```
Comment (orbeckst) [inline _posts/2024-02-01-abstracts-ugm2024:13]: space
```suggestion
MDAnalysis strives to be a diverse and welcoming community for all. While the event will be free to attend, a limited number of travel bursaries are available to enable attendance for those facing financial barriers. If you would like to apply, please follow the prompts on the [abstract submission form][abstracts].
```

File: pages/ugm2024.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ The agenda will be announced in Spring 2024.
 ## Abstracts and Registration
 *Abstract and bursary application deadline: Friday, March 15, 2024*
 
-A call for abstracts will open February 1, 2024. We welcome abstracts from all areas relevant to MDAnalysis's work, including the use and development of open source software and tools for molecular simulations data analysis. 
+You may now submit an abstract using the [abstract submission form](https://forms.gle/usXuZVeM1iZiTDZY8). We welcome abstracts from all areas relevant to MDAnalysis's work, including the use and development of open source software and tools for molecular simulations data analysis.
```
Comment (orbeckst) [inline pages/ugm2024.md:27]: add deadline for abstract submission

**Review by orbeckst — APPROVED** [review]
Minor suggestions — please follow up as you see fit. Thanks a lot for putting it together!

File: pages/ugm2024.md (hunk 3 above)
Comment (jennaswa) [reply pages/ugm2024.md]: It is also listed above this paragraph, but I've added the deadline after the submission form to make it even more clear.

File: _posts/2024-02-01-abstracts-ugm2024 [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+The deadline for submitting your abstract and/or bursary application is **March 15, 2024**. Submissions will be reviewed by the end of March 2024; we will contact applicants shortly after.
```
[the remark this answers is not in the document]
Comment (jennaswa) [reply _posts/2024-02-01-abstracts-ugm2024:15]: Likely not - last year we prioritized people who submitted abstracts. I will add some language to clarify.

**Pull request merged**